infra/capi/cluster-capi-infrastructure-docker/cluster-api-infrastructure-docker.yaml

```diff
@@ -46,6 +41,154 @@ spec:
     singular: dockercluster
   scope: Namespaced
   versions:
+  - deprecated: true
```

Comment:
Trying to understand why L44 to L191 is being added now, seems like this is being set as deprecated as well.

Reply:
Same as above

Reply:
@efiacor Thanks for the response,
For this particular change, what is the corresponding file from https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.8.1 that drove the change? Thanks